packages/edit-site/src/components/post-list/index.js

```diff
@@ -608,13 +291,35 @@ export default function PostList( { postType } ) {
 				fields={ fields }
 				actions={ actions }
 				data={ records || EMPTY_ARRAY }
-				isLoading={ isLoadingMainEntities || isLoadingAuthors }
+				isLoading={ isLoadingMainEntities || isLoading }
 				view={ view }
 				onChangeView={ setView }
 				selection={ selection }
 				onChangeSelection={ onChangeSelection }
 				getItemId={ getItemId }
 				defaultLayouts={ defaultLayouts }
+				header={
+					window.__experimentalQuickEditDataViews &&
+					view.type !== LAYOUT_LIST &&
+					postType === 'page' && (
+						<Button
+							size="compact"
+							isPressed={ quickEdit }
+							icon={ drawerRight }
+							label={
+								! quickEdit
+									? __( 'Show quick edit sidebar' )
+									: __( 'Close quick edit sidebar' )
+							}
+							onClick={ () => {
+								history.push( {
+									...location.params,
+									quickEdit: quickEdit ? undefined : true,
```

Comment:
I have checked and the quickEdit URL parameter seems to be cleared properly when switching screen (back to Design or into the Editor).